jans-linux-setup/jans_setup/setup_app/utils/package_utils.py

```
@@ -52,9 +52,11 @@ def check_and_install_packages(self):
             print(f"{base.os_type} {base.os_version} is not supported.")
             print(f"Supported distributions: {supported}")
             sys.exit(1)
-        
+
         install_command, update_command, query_command, check_text = self.get_install_commands()
```

Ruff (0.15.9)

[warning] Unpacked variable `query_command` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)